tests/installation/bootloader_settings/disable_boot_menu_timeout_right_key.pm

```diff
@@ -0,0 +1,20 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Disable grub timeout from the Installer
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+use testapi;
+
+sub run {
+    $testapi::distri->get_installation_settings()->access_booting_options();
+    send_key_until_needlematch 'inst-bootloader-options-highlighted', 'right';
```

Comment:
for this case you need to wrap this in a method and put testapi in Controller, it is the exception of course.